v2/tools/generator/internal/astmodel/testdata/TestConversionFunctionBuilderBuildConversion_GivenSourceAndDestinationTypes_GeneratesExpectedCode/from_enum_to_string.golden

```diff
@@ -0,0 +1,8 @@
+package TestCase
+
+func TestCase() {
+
+	// Generated by TestConversionFunctionBuilderBuildConversion_GivenSourceAndDestinationTypes_GeneratesExpectedCode/from_enum_to_string
+	destination = string(source)
+
```

Comment:
minor, some weird whitespace here

Reply:
This is deliberate - so the code generated by the statement is visually separated from the boilerplate injected so that dst will generate output.